crates/nu-command/src/core_commands/echo.rs

```diff
@@ -0,0 +1,55 @@
+use nu_protocol::ast::Call;
+use nu_protocol::engine::{Command, EngineState, Stack};
+use nu_protocol::{Example, PipelineData, ShellError, Signature, SyntaxShape, Value};
+
+#[derive(Clone)]
+pub struct Echo;
+
+impl Command for Echo {
+    fn name(&self) -> &str {
+        "echo"
+    }
+
+    fn usage(&self) -> &str {
+        "Echo the arguments back to the user."
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build("echo").rest("rest", SyntaxShape::Any, "the values to echo")
+    }
+
+    fn run(
+        &self,
+        _engine_state: &EngineState,
+        _stack: &mut Stack,
+        _call: &Call,
+        input: PipelineData,
+    ) -> Result<PipelineData, ShellError> {
+        Ok(input)
```

Comment:
Apologies, looks like I misspoke a little. The arguments from the `rest` argument are what should be turned into the output.

Reply:
No problem!
I'll deal with it in some minutes! Thanks for the feedback!